eventarc_gke/scripts/deploy_gke_service.sh

```diff
@@ -0,0 +1,15 @@
+# [START terraform_eventarc_deploy_container]
```

Comment:
The terraform-docs-samples repository doesn't allow CLIs. Please remove this file from the PR.  

Can you include these commands directly in your cgc doc  instead of using a GitHub include.?

Reply:
Yes, updated the commit files. Thanks, Betsy!